### PR TITLE
Resolves #2110 - Dialog names all Dialog1

### LINKED ISSUE #2110: Dialog Name Change for Project Bowie
Review all of the following scripts:

- [x] Actions - New Job Reported
- [x] Actions - Send SVES
- [x] Notes - AREP Form Receivied
- [x] Notes - Baby Born
- [ ] Notes - Change Report Form Received
- [x] Notes - Deceased Client Summary
- [x] Notes - HH Comp Change
- [x] Notes - Incarceration
- [x] Notes - LTC - 1503
- [x] Notes - LTC - 5181
- [x] Notes - Pregnancy Reported
- [x] Action - ABAWD FSET Exemption Checker
- [x] Notes - Verifications Needed

Need to determine if there is more than one dialog defined within the script. If there is then the dialog needs to be changed and it also needs to be moved to be defined in line with the code - right before the dialog.

I am doing this.

### PR DESCRIPTION
**Pull request by C-Love**
Blip - 'Altering scripts for the Changes Reported workflow so there is no dialog confusion.'

### CODE REVIEW

**Comment by theVKC** [comment]
So @C-Love, you're doing a `dialog dialog1` with these, when I assumed it would simply be `dialog`. I think that avoids dialog IDs. It's also less code. :P

Can you modify these for me?

**Comment by C-Love** [comment]
I am running into weird errors when I name them just 'Dialog'. Do you want to test? Basically, an empty dialog starts, that I need to close, then out of range dialog errors happen and dialogs will run more than once.

**Comment by theVKC** [comment]
Don't name the dialog dialog. Just use the dialog function without passing a parameter.

**Comment by C-Love** [comment]
Ok - it feels so weir BUT it appears to be working. Correct as usual King Friday.

**Comment by theVKC** [comment]
Now the only problem is a merge conflict. Can you fetch upstream and attempt to resolve it, @C-Love?

**Comment by theVKC** [comment]
So @C-Love: the dialogs should be called `Dialog1`, and the dialog function (which displays the dialog) should just read `dialog`. I'll comment the first one specifically with how the lines should look.

With the merge conflict, we might run into problems on this one. Let me know how you want to resolve it. :/

File: Script Files/ACTIONS/ACTIONS - ABAWD FSET EXEMPTION CHECK.vbs [hunk 1]
```diff
@@ -53,7 +53,7 @@ STATS_denomination = "M"       		'M is for each MEMBER
 'END OF stats block=========================================================================================================
 
 'DIALOGS-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-BeginDialog case_number_dialog, 0, 0, 166, 70, "Case number dialog"
+BeginDialog , 0, 0, 166, 70, "Case number dialog"
```
Comment (theVKC) [inline Script Files/ACTIONS/ACTIONS - ABAWD FSET EXEMPTION CHECK.vbs]: `BeginDialog Dialog1, 0, 0, 166, 70, "Case number dialog"`

File: Script Files/ACTIONS/ACTIONS - ABAWD FSET EXEMPTION CHECK.vbs [hunk 2]
```diff
@@ -86,7 +86,7 @@ cstr(footer_month)
 
 DO
 	err_msg = ""
-	DIALOG case_number_dialog
+	DIALOG
```
Comment (theVKC) [inline Script Files/ACTIONS/ACTIONS - ABAWD FSET EXEMPTION CHECK.vbs]: This is correct.

**Comment by theVKC** [comment]
@C-Love, since this got so kittywampus I'm going to close this pull request and we can revisit when you return. I'm off Friday-Tuesday so I've delayed Bowie until the June release.

**Pull request closed without merging**